### PR TITLE
Match PR directory links.

### PR DESCRIPTION
**Pull request by fejta**
Presubmits tend to get uploaded in a `gs://prefix/directory/job/1234.txt` with a link to `gs://prefix/pull/123/job/1234`

### CODE REVIEW

File: pkg/updater/pubsub.go [hunk 1]
```diff
@@ -214,10 +214,14 @@ func processNotification(paths map[gcs.Path][]string, n *pubsub.Notification) ([
 	base := path.Base(obj)
 	dur, ok := namedDurations[base]
 	if !ok { // Maybe it is a junit file, should finish soon
-		if !strings.HasPrefix(base, "junit") || !strings.HasSuffix(base, ".xml") {
+		switch {
+		case strings.HasPrefix(base, "junit") && strings.HasSuffix(base, ".xml"):
+			dur = 5 * time.Minute
+		case strings.HasSuffix(base, ".txt") && strings.Contains(obj, "directory/"):
```
Comment (michelle192837) [inline pkg/updater/pubsub.go:220]: nit: Consider a comment for what the new case covers (I keep forgetting these are symlinks?)

**Review by michelle192837 — APPROVED** [review]
/hold

File: pkg/updater/pubsub.go (hunk 1 above)
Comment (fejta) [reply pkg/updater/pubsub.go:220]: Done

**Pull request merged**